docker/k8s/README.md

```diff
@@ -36,21 +28,26 @@ in SGX-SIM mode.
     ```bash
     kubectl create -f sgx-enclave-manager-deployment.yaml
     ```
-5.  Start `TCS Listener` (to accept and handle requests from clients)
+3.  Start `http jrpc Listener service` (to accept and handle http jrpc requests from clients)
     ```bash
-    kubectl create -f tcs.yaml
+    kubectl create -f listener.yaml
     ```
 
 ## Propose a request
-1. Send the request 
+1. Before sending a workload request check if sharedkv, enclave manager and listener pods are running
+   ```bash
+    kubectl get pods
+    ```
+
+2. Send the request 
     ```bash
     kubectl create -f propose-requests-job.yaml
     ```
 
     > The above command would run the 
     > [propose_requests.sh](./scripts/propose_requests.sh) as a kubernetes job
 
-2. Check the request handling logs (the request should be done in several 
+3. Check the request handling logs (the request should be done in several 
```

Comment:
How do we run other workload for example echo using generic client? Getting into avalon shell and running generic command would be nicer approach..

Reply:
This is an example kubernetes job pod to create and run a workloads automatically without user intervention. In this case the job pod runs heart-disease workload.  If we use kubernetes pod to create a shell then user has to manually get into the pod container. We can do this as a separate Kubernetes deployment file in a different PR. But purpose of job pods is is to runs jobs (workloads) to completion.  https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/.